PocketKit/Tests/SyncTests/Support/Space+factories.swift

```diff
@@ -110,8 +110,6 @@ extension Space {
     }
 }
 
-
-
```

Comment:
just another note, should we enforce SwiftLint rules instead of having them as warnings? I notice at times we merge them in

Reply:
@cyndichin yes we should.